erpnext/accounts/doctype/payment_ledger_entry/payment_ledger_entry.py

```diff
@@ -18,5 +31,119 @@ def validate_account(self):
 		if not valid_account:
 			frappe.throw(_("{0} account is not of type {1}").format(self.account, self.account_type))
 
+	def validate_account_details(self):
+		"""Account must be ledger, active and not freezed"""
+
+		ret = frappe.db.sql(
+			"""select is_group, docstatus, company
+			from tabAccount where name=%s""",
+			self.account,
+			as_dict=1,
+		)[0]
```

Comment:
- why raw sql? 
- `ret` is bad variable name 😅, `account` or `account_data` adds more info.